HHS/OPRE-OPS
# Task 844

## What changed

Adds Project / Agreement Summary Card

## Issue

#844 

## How to test

Select and Project and Agreement on step 1, and then see the summary card on step 2.

## Screenshots

<img width="995" alt="image" src="https://user-images.githubusercontent.com/4629398/232594865-76fd22d5-c1ca-4547-8052-6a97e72732e1.png">


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
